sql-files/upgrades/upgrade_20200615.sql

```diff
@@ -0,0 +1 @@
+INSERT IGNORE `sc_data` (`account_id`, `char_id`, `type`, `tick`) SELECT `account_id`, `char_id`, '746', '-1' FROM `char` WHERE `option` & '4194304' != '0';
```

Comment:
Make sure to check if 746 is still correct before merging, since we always add SCs